modules/theme/src/theme-core/definitions/apps/user-portal.less

```diff
@@ -163,5 +163,17 @@
     width: 100%;
 }
 
+/*****************************************
+             User sessions
+*****************************************/
+
+.user-sessions-edit-secion {
+    .ui.table td {
+        &.app-name {
+            padding-left: 0;
+        }
+    }
+}
+
```

Comment:
There are two empty lines. We can remove one?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2906/commits/775e14068866660fb0e09d7defe813b4f3d87df7